ui/app/AppLayouts/Profile/panels/TokenSettingsModalContent.qml

```diff
@@ -0,0 +1,197 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+
+import utils 1.0
+
+import StatusQ.Controls 0.1
+
+import shared.panels 1.0
+import shared.popups 1.0
+import shared.controls 1.0
+
+Item {
+    id: modalBody
+
+    property var defaultTokenList
+    property var customTokenList
+    signal toggleVisibleClicked(int chainId, string symbol)
+    signal removeCustomTokenTriggered(int chainId, string address)
+    signal showTokenDetailsTriggered(int chainId, string address, string name, string symbol, string decimals)
+
+    SearchBox {
+        id: searchBox
+        customHeight: 36
+        fontPixelSize: 12
+        anchors.top: modalBody.top
+    }
+
+    Component {
+        id: tokenComponent
+        Rectangle {
```

Comment:
are we implementing this https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=642%3A63538?

Can we use the existing StatusQ component instead? You can run the demo app and see the list items page. We have things with checkbox.

Reply:
done